packages/App/src/store/saga/walletItemsRequest.tsx

```diff
@@ -33,6 +33,7 @@ export function* getWalletItems(): Generator {
     )) as Array<WalletInfo>;
 
     if (response) {
+      response;
```

Comment:
?)

Reply:
yep)

Reply:
I moved logic from here to UI)and there added card-button, that add new card)

Reply:
That isn't. You are not returning `response`

Reply:
ahh right) there is no need to return actually
I deleted it, and make commit